src/components/MessageList/MessageList.jsx

```diff
@@ -74,7 +74,7 @@ class MessageList extends Component {
                 to="16"
                 values="16; 12; 16;"
                 dur="500ms"
-                repeat="always"
+                restart="always"
```

Comment:
Was that ALL that we had to do to get that warning to go away? does it still restart! This is amaze!